### ISSUE TITLE
Question on running Ordering.API and Swagger UI

### ISSUE THREAD

**Issue by dogaanismail** [dotnet/eShopOnContainers#1904]
Hello,

I have been trying to run Ordering.API and trying to test endpoints. By the way, I have been developing the project from scratch to learn the solution logic. I have a Github repository. You can find below.

[eCommerceOnContainer](https://github.com/dogaanismail/eCommerceOnContainer)

My problem is that I can not run Swagger UI for Ordering.API. How can I run Swagger UI for Ordering.API?

![Screenshot_60](https://user-images.githubusercontent.com/30376425/166483908-7c76c693-42fd-4ac4-a2a0-afc94d8787b8.png)

Here is 
   			{
			  "iisSettings": {
				"windowsAuthentication": false,
				"anonymousAuthentication": true,
				"iisExpress": {
				  "applicationUrl": "http://localhost:55102/",
				  "sslPort": 0
				}
			  },
			  "profiles": {
				"IIS Express": {
				  "commandName": "IISExpress",
				  "launchBrowser": true,
				  "launchUrl": "/swagger",
				  "environmentVariables": {
					"ASPNETCORE_ENVIRONMENT": "Development"
				  }
				},
				"Microsoft.eCommerceOnContainers.Services.Ordering.API": {
				  "commandName": "Project",
				  "launchBrowser": true,
				  "launchUrl": "http://localhost:55102/",
				  "environmentVariables": {
					"ASPNETCORE_ENVIRONMENT": "Development"
				  }
				}
			  }
			}``

**Comment by onurkanbakirci** [comment]
Hi @dogaanismail . Did you try `localhost:55102/swagger`? İf you use something like:
```
....
"Microsoft.eCommerceOnContainers.Services.Ordering.API": {
      "commandName": "Project",
      "dotnetRunMessages": true,
      "launchBrowser": true,
      "launchUrl": "swagger",
      "applicationUrl": "http://localhost:55102",
      "environmentVariables": {
        "ASPNETCORE_ENVIRONMENT": "Development"
      }
    },
....
```
in launchSettings.json file, swagger tab will automatically open in default browser.

**Comment by dogaanismail** [comment]
Hello @onurkanbakirci,

After long efforts, I have just solved the problem. In Program.cs, I had to comment some lines that are about Grpc ports. (I will be resourcing using of Grpc later by the way.) 
In addition, I have edited launchSettings.json file with your instructions. I have successfully run Swagger UI :) 

Thank you for your support.
Best regards


![Screenshot_65](https://user-images.githubusercontent.com/30376425/167229077-86b1ac21-f0a0-4a4d-b9cb-86a6802b1ff4.png)
